examples/hybrid/cli_options.jl

```diff
@@ -129,8 +125,11 @@ function parse_commandline()
         default = :none
         "--density_upwinding"
         help = "Denisity upwinding mode [`none` (default), `first_order` , `third_order`, `boris_book`, `zalesak`]"
-        arg_type = Symbol
         default = :none
+        "--edmfx_upwinding"
+        help = "EDMFX upwinding mode [`none` (default), `first_order` , `third_order`, `boris_book`, `zalesak`]"
+        arg_type = Symbol
+        default = :none # TODO: change to :first_order (or higher?)
```

Comment:
When we say `none` does it mean we are using central advection scheme?

Reply:
yep